runtime/default.policy

```diff
@@ -31,6 +54,43 @@ macros:
 # Rules determine what the Datadog security-agent will monitor.#
 ################################################################
 rules:
+## Process Activity Rules ##
+  - id: potential_web_shell
+    description: Potential webshell execution
+    expression: >-
+      (exec.filename in SHELLS || exec.filename in ["/usr/bin/curl", "/usr/bin/wget"] || exec.filename in SHELL_UTILS) && process.ancestors.name in WEBAPP_PROCESSES || process.ancestors.name =~ "php*"
```

Comment:
it should be I think
```
(exec.filename in SHELLS || exec.filename in ["/usr/bin/curl", "/usr/bin/wget"] || exec.filename in SHELL_UTILS) && (process.ancestors.name in WEBAPP_PROCESSES || process.ancestors.name =~ "php*")
```

with parentheses for ancestors otherwise `|| process.ancestors.name =~ "php*"` will be applied outside of the previous `&&`